src/setup.erl

```diff
@@ -61,7 +61,46 @@ has_cluster_system_dbs([Db|Dbs]) ->
     end.
 
 enable_cluster(Options) ->
-    enable_cluster_int(Options, is_cluster_enabled()).
+    case proplists:is_defined(remote_node, Options) of
```

Comment:
If Options is a true `{K,V}` list, use `lists:keyfind` instead of `proplists:is_defined`. If you assume that Options may contain just `remote_node` with no associated value, then ignore this comment.